Final Project/Portfolio/404.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
```

Comment:
Good call on including a 404 page! You may want to share the same *head* from the other HTML documents, and move some of these inline styles into your CSS as well. Then you'll have more control and it can feel like a seamless experience.